app/src/main/java/preston/com/stockapp/HomeActivity.java

```diff
@@ -0,0 +1,68 @@
+package preston.com.stockapp;
+
+import android.graphics.Typeface;
+import android.os.Bundle;
+import android.support.v4.app.FragmentManager;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.widget.TextView;
+
+import preston.com.stockapp.util.LoginFragment;
+import preston.com.stockapp.util.SignUpFragment;
+
+/**
+ * Created by Alex Preston on 9/5/16.
+ */
+public class HomeActivity extends AppCompatActivity {
+
+    private TextView welcome;
+    private TextView signUp;
+    private TextView login;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.home_activity);
+
+        welcome = (TextView) findViewById(R.id.welcome_home);
+        signUp = (TextView) findViewById(R.id.sign_up_home);
+        login = (TextView) findViewById(R.id.login_home);
+
+        Typeface face = Typeface.createFromAsset(getAssets(), "bondini.ttf");
+        welcome.setTypeface(face);
+        signUp.setTypeface(face);
+        login.setTypeface(face);
+
+        signUp.setOnClickListener(new View.OnClickListener() {
```

Comment:
Just implement the onClickListener on the activity. Use a switch(v.getId) to distinguish the specific views